### PR TITLE
Upgrade opponent robots

### PR DESCRIPTION
**Pull request by VladimirVincan**
TODO:

1. - [x] correct robot height

image
2. maybe align the robots to the start line?

Added ArUco tags to robots. 
Enabled color swapping.
Aligned opponent robots.
Fixed Viewpoint position from:
- position 0.016513897535666677 0.2116180370445013 10.843371189226236

to:
- position 0 0 10

image
image
image

### CODE REVIEW

**Review by MarijaGolubovic — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -1,5 +1,5 @@
 *.bin binary
 *.obj binary
-*.png binary
+*.png filter=lfs diff=lfs merge=lfs -text
```
Comment (lukicdarkoo) [inline .gitattributes:3]: Sta je ovo? :)

Comment (lukicdarkoo) [reply .gitattributes:3]: Msm da si vratio LFS. Problem sa LFS-om je bio ogranicen protok (GitHub dozvoljava samo 1GB/m), pa smo ga morali ukloniti

File: mep3_simulation/webots_data/protos/OpponentBox.proto [hunk 2]
```diff
@@ -1,56 +1,114 @@
...
 
 PROTO OpponentBox [
   field SFString   name         "opponent_box"
   field SFVec3f    translation  0 0 0
   field SFRotation rotation     0 1 0 0
   field SFString   controller   "opponent_robot"
+  field SFString {"yellow", "purple"} color "purple"
+  field SFInt32    aruco_number 1
+  hiddenField SFColor yellow_vec 0.972549 0.694118 0.0666667
+  hiddenField SFColor purple_vec 0.360784 0.282353 0.541176
```
Comment (lukicdarkoo) [inline mep3_simulation/webots_data/protos/OpponentBox.proto:12]:
```suggestion
  field SFInt32    arucoNumber 1
  hiddenField SFColor yellowVector 0.972549 0.694118 0.0666667
  hiddenField SFColor purpleVector 0.360784 0.282353 0.541176
```
U Webots-u koriste camel-case bez skracenica pa mozemo i mi tako

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
Odlicno!

Samo nesto cudno se desilo sa LFS-om

File: mep3_simulation/webots_data/protos/OpponentBox.proto [hunk 3]
```diff
@@ -1,56 +1,114 @@
...
-					}
-				]
-			}
-		]
+    children [
+      Transform {
+        translation 0 0 0.385
+        children [
+          Transform {
+            translation 0 0 0.01
```
Comment (lukicdarkoo) [inline mep3_simulation/webots_data/protos/OpponentBox.proto:38]: Mozda da se podigne jos samo malo, sada sam primijetio da se desava z-fighting ponekad

**Review by lukicdarkoo — APPROVED** [review]

**Pull request merged**